test/cri-containerd/main.go

```diff
@@ -282,9 +284,13 @@ func pullRequiredImagesWithLabels(t *testing.T, images []string, labels map[stri
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	sb := &runtime.PodSandboxConfig{
-		Labels: labels,
+	sb := &runtime.PodSandboxConfig{}
+	for _, o := range opts {
+		if err := o(sb); err != nil {
+			t.Errorf("failed to apply PodSandboxConfig option: %s", err)
```

Comment:
Maybe `Fatalf` here so the pulling of the images doesn't occur if the opt applies failed.

Reply:
From reading the current options there's no where that they can error but obviously who knows what the future holds 🤣

Reply:
yeah, I agree. it's better to fail fast :)